src/main/java/com/attacktimer/AttackTimerMetronomePlugin.java

```diff
@@ -347,7 +347,7 @@ private int getWeaponSpeed(int weaponId, PoweredStaves stave, AnimationData curA
         return adjustSpeedForLeaguesIfApplicable(speed); // Deadline for next available attack.
     }
 
-    private static final int TARGET_DUMMY_ID = 10507;
+    private static final List<Integer> SPECIAL_NPCS = Arrays.asList(10507, 9435, 9438, 9441, 9444); // Combat Dummy + Nightmare Pillars
```

Comment:
Maybe worth moving the special npcs into its own enum/file.

Reply:
Worth for future updates, I think we can leave it like that meanwhile